python/lsst/afw/display/interface.py

```diff
@@ -29,6 +29,9 @@
 import importlib
 import lsst.afw.geom  as afwGeom
 import lsst.afw.image as afwImage
+import lsst.log
+
+logger = lsst.log.Log.getLogger("afw.display.interact")
```

Comment:
This is a typo. It should have been 
`logger = lsst.log.Log.getogger("afw.display.interface")`.

If lsst.log is like the logging module, this can matter a great deal. For example, if the "afw" logger is set to a given level (logging.INFO for example), then "afw.display.interface" will inherit that setting. For this reason other projects that I've been involved in use this convention, although I have also seen it become more specific ("afw.display.interface.interact") in a function if there is a reason that the function might want to be singled out explicitly. I don't know enough about the C++ logger and what effect this might have, but if it has similar functionality I would recommend that we adopt a similar convention. Even if it isn't, it might still be useful for the python side of the code.